pyquidax/wrappers/deposits.py

```diff
@@ -33,3 +33,26 @@ def get_by_id(self, deposit_id: str, user_id: str = "me"):
             url=f"{self.base_url}/users/{user_id}/deposits/{deposit_id}",
             method=HTTPMethod.GET,
         )
+
+class Deposit(BaseAsyncAPIWrapper):
```

Comment:
The name of the class should be `AsyncDeposit` you can't have two classes bearing the same name in the same module.

Reply:
When I request for changes and you've resolved them please mark as resolved